cli/azd/pkg/ux/prompt.go

```diff
@@ -44,6 +44,8 @@ type PromptOptions struct {
 	ClearOnCompletion bool
 	// Whether or not to capture hint keys (default: true)
 	IgnoreHintKeys bool
+	// The optional second line message to display (default: "")
+	SecondLineMessage string
```

Comment:
I feel like we should be more descriptive about what the intent of this field is... it is basically help text, right? 
We should have a single way of representing help text... we either show it always or on "?" hint key press.